install.sh

```diff
@@ -80,19 +80,19 @@ _check_dependencies() {
     posix_check_dependencies="${1:-0}"
     unset error_list warning_list
 
-    for program in curl find xargs mkdir rm grep sed; do
+    for program in curl find xargs mkdir rm grep sed sleep ps; do
         command -v "${program}" 2>| /dev/null 1>&2 || error_list="${error_list}\n${program}"
     done
 
     { ! command -v file && ! command -v mimetype; } 2>| /dev/null 1>&2 &&
         error_list="${error_list}\n\"file or mimetype\""
 
     [ "${posix_check_dependencies}" != 0 ] &&
-        for program in awk cat date ps sleep; do
+        for program in awk cat date; do
             command -v "${program}" 2>| /dev/null 1>&2 || error_list="${error_list}\n${program}"
         done
 
-    for program in ps tail; do
+    for program in tail; do
```

Comment:
Codacy found an issue: [This loop will only ever run once. Bad quoting or missing glob/expansion?](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6363604)